capstone/src/main/java/com/google/sps/Objects/response/BasicMemberResponse.java

```diff
@@ -0,0 +1,79 @@
+package com.google.sps.Objects.response;
+
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.Entity;
+
+/**
+ * Contains information for the response sent by AllGroupMembersServlet.
+ */
+public final class BasicMemberResponse {
+
+  private final String profilePic;
+  private final String firstName;
+  private final String lastName;
+  private final String userId;
+
+  /**
+   * Constructs a BasicMemberResponse object.
+   *
+   * @param profilePic String url of profile image 
+   * @param firstName String of user first name 
+   * @param lastName String of user last name 
+   * @param userId user id 
+   */
+  public BasicMemberResponse(String profilePic, String firstName, String lastName,String userId) {
+    this.profilePic = profilePic;
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.userId = userId;
+  }
+  
+  /**
+   * Constructs a BasicMemberResponse object from an entity
+   *
+   * @param profilePic String url of profile image 
+   * @param firstName String of user first name 
+   * @param lastName String of user last name  
+   * @param userId user id 
+   */
+  public static BasicMemberResponse fromEntity(Entity entity) {
```

Comment:
I haven't played around with this and you guys have, but I have a question: Seeing as `BasicMemberResponse` appears to be the same as `MemberResponse` minus the badges, if you don't set any badges, can you avoid sending `badges: null/undefined/[]` while using a MemberResponse instead of a BasicMemberResponse?

I'd hate to see you have to create a new version of the exact same object every time you don't want to include a field.

I would still maybe even suggest using MemberResponse with `badges: null/undefined/[]/[actual, badges, here]` instead of having to create a new BasicMemberResponse when the two are so similar. Thoughts?

Reply:
I think sending null for badges and getting rid of BasicMemberResponse makes sense. Why do you say "can you avoid sending badges: null/undefined/[] while using a MemberResponse"

Reply:
I was trying to avoid sending unnecessary info, but it makes sense to remove BasicMemberResponse and set null for Badges.